passkey-client/src/lib.rs

```diff
@@ -209,6 +212,14 @@ where
                 None
             };
 
+        let uv = matches!(
+            request
+                .authenticator_selection
+                .map(|s| s.user_verification)
+                .unwrap_or_default(), // Default is Preferred
```

Comment:
Small nit: If we're going to add a comment specifying the default, would it make more sense to instead do:
```rust
.unwrap_or(UserVerificationRequirement::Preferred)
```

I also wonder if this condition is better expressed in reverse:
```rust
let uv = request.authenticator_selection.map(|s| s.user_verification)
        != Some(UserVerificationRequirement::Discouraged);
```

Reply:
Yes, much better with the inversion! I got stuck on learning `matches!` and then everything looked like a nail for that hammer :)